src/layman/common/prime_db_schema/publications_test.py

```diff
@@ -168,43 +168,43 @@ class TestSelectPublicationsComplex:
          {'title': 'Jednobodová vrstva Kodaň',
           'access_rights': {'read': {settings.RIGHTS_EVERYONE_ROLE},
                             'write': {settings.RIGHTS_EVERYONE_ROLE}},
-          'bbox': (1843000, 6323000, 1843000, 6323000),
-          'crs': 'EPSG:3857',
+          'bbox': (16.55595068632278, 49.28289621550056, 16.55595068632278, 49.28289621550056),
+          'crs': 'EPSG:4326',
```

Comment:
I prefer to keep original test cases in the left, e.g. `'bbox': (1843000, 6323000, 1843000, 6323000), 'crs': 'EPSG:3857',`, and to add new test cases for other CRSs with new layer names. For original test cases, publication names make sense, but for new ones it's not so fitting.

Maybe we can create brand new test in this class only for bbox in different CRS. We can discuss it tomorrow.